app/javascript/packs/onboardingRedirectCheck.jsx

```diff
@@ -10,17 +23,17 @@ HTMLDocument.prototype.ready = new Promise((resolve) => {
 
 function redirectableLocation() {
   return ![
-    '/onboarding',
-    '/signout_confirm',
-    '/privacy',
+    routeForOnboarding,
+    routeForSignoutConfirm,
+    routeForPrivacy,
     '/admin/creator_settings/new',
   ].includes(window.location.pathname);
 }
 
 function redirectableCreatorOnboardingLocation() {
   return (
     redirectableLocation() &&
-    !['/code-of-conduct', '/terms'].includes(window.location.pathname)
```

Comment:
For `signout_confirm`, I have used `signoutConfirmPath()` and for `code-of-conduct` I have used `codeOfConductPath ()`. 
I am confused why both of these are working (tests-passing) because one uses `_` and other uses `-` as separator.


Also, what should I write for `/admin/creator_settings/new`, I am not sure about this.